src/main/java/org/folio/des/service/JobService.java

```diff
@@ -40,6 +40,11 @@ public interface JobService {
    */
   void deleteOldJobs();
 
+  /**
+   * Resend export file.
+   */
+  void resend(Job job);
```

Comment:
Lets make name consistent with download functionality, so name it as resendExportedFile

Reply:
Also downloadExportedFile accepts UUID, so lets make resend method consistent, it also shold accept UUID

Reply:
fixed